pegasus/src/database.rb

```diff
@@ -59,7 +59,7 @@ def no_credit_count
 end
 
 def credit_count
-  DB[:cdo_state_promote].where(cs_counts_t: 'Yes').count
+  DB[:cdo_state_promote].where("cs_counts_t = 'Yes' || cs_counts_t = 'Other'").count
```

Comment:
It would be neat if there were some syntax that let you list all the possibilities, but if this is the best one, then all good.